website/app.py

```diff
@@ -21,8 +22,12 @@
 import website.models
 from website.routes import make_url_map
 from website.addons.base import init_addon
-from website.project.model import ensure_schemas
+from website.project.model import ensure_schemas, Node
 
+def build_js_config_files(settings):
+    with open(os.path.join(settings.STATIC_FOLDER, 'js', 'nodeCategories.json'), 'wb') as fp:
```

Comment:
It might make sense to dump this to `public` or explicitly add it to `.gitignore` so it doesn't get added to source control.